modules/bidViewability.md

```diff
@@ -0,0 +1,48 @@
+# Overview
+
+Module Name: bidViewability
+
+Purpose: Track when a bid is viewable
+
+Maintainer: harshad.mane@pubmatic.com
+
+# Description
+- This module, when included, will trigger a BID_VIEWABLE event which can be consumed by Analytics adapters, bidders will need to implement `onBidViewable` method to capture this event
+- GPT API is used to find when a bid is viewable, https://developers.google.com/publisher-tag/reference#googletag.events.impressionviewableevent . This event is fired when an impression becomes viewable, according to the Active View criteria.
+Refer: https://support.google.com/admanager/answer/4524488
+- The module does not work with adserver other than GAM with GPT integration
+- Logic used to find a matching pbjs-bid for a GPT slot is ``` (slot.getAdUnitPath() === bid.adUnitCode || slot.getSlotElementId() === bid.adUnitCode) ``` this logic can be changed by using param ```customMatchFunction```
+- When a rendered PBJS bid is viewable the module will trigger BID_VIEWABLE event, which can be consumed by bidders and analytics adapters
+- For the viewable bid if ```bid.vurls type array``` param is and module config ``` firePixels: true ``` is set then the URLs mentioned in bid.vurls will be executed. Please note that GDPR and USP related parameters will be added to the given URLs, here we have assumed that URLs will always have "?" symbol included.
+
```

Comment:
Is there a way for the bidder to tell if viewability module is present?  I ask because viewability stats aren't useful unless you know if the inventory is measurable or not.

Reply:
bidder's onBidViewable handler will trigger only when this module is present. 
We can add a boolean flag in PBJS name-space to let bidders know if the module is added or not.

Reply:
Hello @mike-chowla 

I have made changes in module config, now we need to set enabled flag to true to enable the module.
Bidders can check this flag to know whether the module is present or not.
```
pbjs.setConfig({
        bidViewability: {
            enabled: true
        }
    });
```